third_party/accessibility/build.gn

```diff
@@ -0,0 +1,174 @@
+    "ax/ax_action_handler_base.cc",
+    "ax/ax_action_handler_base.h",
+    "ax/ax_base_export.h",
+    "ax/ax_clipping_behavior.h",
+    "ax/ax_constants.h",
+    "ax/ax_coordinate_system.h",
+    "ax/ax_enum_util.cc",
+    "ax/ax_enum_util.h",
+    "ax/ax_enums.h",
+    "ax/ax_event_generator.cc",
+    "ax/ax_event_generator.h",
+    "ax/ax_event_intent.cc",
+    "ax/ax_event_intent.h",
+    "ax/ax_export.h",
+    "ax/ax_mode.cc",
+    "ax/ax_mode.h",
+    "ax/ax_mode_observer.h",
+    "ax/ax_node.cc",
+    "ax/ax_node.h",
+    "ax/ax_node_data.cc",
+    "ax/ax_node_data.h",
+    "ax/ax_node_position.cc",
+    "ax/ax_node_position.h",
+    "ax/ax_node_text_styles.cc",
+    "ax/ax_node_text_styles.h",
+    "ax/ax_offscreen_result.h",
+    "ax/ax_position.h",
+    "ax/ax_range.h",
+    "ax/ax_relative_bounds.cc",
+    "ax/ax_relative_bounds.h",
+    "ax/ax_role_properties.cc",
+    "ax/ax_role_properties.h",
+    "ax/ax_table_info.cc",
+    "ax/ax_table_info.h",
+    "ax/ax_tree.cc",
+    "ax/ax_tree.h",
+    "ax/ax_tree_data.cc",
+    "ax/ax_tree_data.h",
+    "ax/ax_tree_id.cc",
+    "ax/ax_tree_id.h",
+    "ax/ax_tree_id_registry.cc",
+    "ax/ax_tree_id_registry.h",
+    "ax/ax_tree_manager.h",
+    "ax/ax_tree_manager_map.cc",
+    "ax/ax_tree_manager_map.h",
+    "ax/ax_tree_observer.cc",
+    "ax/ax_tree_observer.h",
+    "ax/ax_tree_update.h",
+  ]
+
+  if (is_mac) {
+    sources += [
+      "ax/platform/ax_platform_node_mac.h",
+      "ax/platform/ax_platform_node_mac.mm",
+    ]
+  }
+
+  public_deps = [
+    "//flutter/fml",
+    "//third_party/skia",
+  ]
+
+  public_configs = [
+    ":accessibility_config",
+    "//flutter:config",
+  ]
+
+  deps = [ "//third_party/skia" ]
+
+  if (is_mac) {
+    libs = [
+      "AppKit.framework",
+      "CoreFoundation.framework",
+      "CoreGraphics.framework",
+      "CoreText.framework",
+      "IOSurface.framework",
+    ]
+  }
+}
+
+if (enable_unittests) {
+  test_fixtures("accessibility_fixtures") {
+    fixtures = []
+  }
+
+  executable("accessibility_unittests") {
+    testonly = true
+
+    sources = [
+      "ax/ax_enum_util_unittest.cc",
+      "ax/ax_event_generator_unittest.cc",
+      "ax/ax_node_data_unittest.cc",
+      "ax/ax_node_position_unittest.cc",
+      "ax/ax_range_unittest.cc",
+      "ax/ax_role_properties_unittest.cc",
+      "ax/ax_table_info_unittest.cc",
+      "ax/ax_tree_unittest.cc",
+      "ax/test_ax_node_helper.cc",
+      "ax/test_ax_node_helper.h",
+      "ax/test_ax_tree_manager.cc",
+      "ax/test_ax_tree_manager.h",
+    ]
+
+    sources += [
+      "ax/platform/ax_platform_node_base_unittest.cc",
+      "ax/platform/ax_platform_node_unittest.cc",
+      "ax/platform/ax_platform_node_unittest.h",
+      "ax/platform/ax_unique_id_unittest.cc",
+      "ax/platform/test_ax_node_wrapper.cc",
+      "ax/platform/test_ax_node_wrapper.h",
+    ]
+
+    deps = [
+      ":accessibility",
+      ":accessibility_fixtures",
+      "//flutter/testing",
```

Comment:
This is also an internal engine dependency.

Reply:
For unit test only target, do we need to worry about this?